src/components/shared/card/index.tsx

```diff
@@ -71,15 +71,17 @@ const Card = ({
       onClick={onClick}
     >
       <div className='flex min-w-0 gap-4'>
-        <div className='flex flex-col'>
-          <CardThumbnail
-            hasThumbnail={hasThumbnail}
-            thumbnail={thumbnail}
-            title={title}
-            onError={() => setImageError(true)}
-          />
-
-          <CardProfile nickName={nickName} profileImage={profileImage} />
+        <div className='flex flex-col justify-between'>
```

Comment:
```suggestion
        <div className='flex-col-between'>
```
제가 만든 TW util 보시면 flex-center / flex-between / flex-col-center / flex-col-between 있습니다.
styles/layout.css 가서 보세용